subsys/bluetooth/mesh/shell/brg_cfg.c

```diff
@@ -0,0 +1,241 @@
+		return -ENOEXEC;
+	}
+
+	shell_print(sh, "Bridged Subnets List:");
+	shell_print(sh, "\tfilter: %02x", rsp.net_idx_filter.filter);
+	shell_print(sh, "\tnet_idx: %04x", rsp.net_idx_filter.net_idx);
+	shell_print(sh, "\tstart_idx: %u", rsp.start_idx);
+	if (rsp.list) {
+		uint16_t net_idx1, net_idx2;
+		int i = 0;
+
+		while (rsp.list->len) {
+			key_idx_unpack_pair(rsp.list, &net_idx1, &net_idx2);
+			shell_print(sh, "\tEntry %d:", i++);
+			shell_print(sh, "\t\tnet_idx1: 0x%04x, net_idx2: 0x%04x", net_idx1,
+				    net_idx2);
+		}
+	}
+	return 0;
+}
+
+static int cmd_bridging_table_get(const struct shell *sh, size_t argc, char *argv[])
+{
+	uint16_t net_idx1, net_idx2, start_idx;
+	struct bt_mesh_bridging_table_list rsp;
+	int err = 0;
+
+	net_idx1 = shell_strtoul(argv[1], 0, &err);
+	net_idx2 = shell_strtoul(argv[2], 0, &err);
+	start_idx = shell_strtoul(argv[3], 0, &err);
+	if (err) {
+		shell_warn(sh, "Unable to parse input string argument");
+		return err;
+	}
+
+	err = bt_mesh_brg_cfg_cli_bridging_table_get(bt_mesh_shell_target_ctx.net_idx,
+						     bt_mesh_shell_target_ctx.dst, net_idx1,
+						     net_idx2, start_idx, &rsp);
+	if (err) {
+		shell_error(sh, "Failed to send Bridging Table Get (err %d)", err);
+		return -ENOEXEC;
+	}
+
+	if (rsp.status) {
+		shell_print(sh, "Bridging Table Get failed with status 0x%02x", rsp.status);
+	} else {
+		shell_print(sh, "Bridging Table List:");
+		shell_print(sh, "\tstatus: %02x", rsp.status);
+		shell_print(sh, "\tnet_idx1: %04x", rsp.net_idx1);
+		shell_print(sh, "\tnet_idx2: %04x", rsp.net_idx2);
+		shell_print(sh, "\tstart_idx: %u", rsp.start_idx);
+		if (rsp.list) {
+			uint16_t addr1, addr2;
+			uint8_t directions;
+			int i = 0;
+
+			while (rsp.list->len) {
+				addr1 = net_buf_simple_pull_le16(rsp.list);
+				addr2 = net_buf_simple_pull_le16(rsp.list);
+				directions = net_buf_simple_pull_u8(rsp.list);
+				shell_print(sh, "\tEntry %d:", i++);
+				shell_print(sh,
+					    "\t\taddr1: 0x%04x, addr2: 0x%04x, directions: 0x%02x",
+					    addr1, addr2, directions);
+			}
+		}
+	}
+	return 0;
+}
+
+SHELL_STATIC_SUBCMD_SET_CREATE(
+	brg_cfg_cmds,
+	SHELL_CMD_ARG(bridge-get, NULL, NULL, cmd_subnet_bridge_get, 1, 0),
+	SHELL_CMD_ARG(bridge-set, NULL, "<State(disable, enable)>", cmd_subnet_bridge_set, 2, 0),
+	SHELL_CMD_ARG(table-size-get, NULL, NULL, cmd_bridging_table_size_get, 1, 0),
+	SHELL_CMD_ARG(table-add, NULL, "<Directions> <NetIdx1> <NetIdx2> <Addr1> <Addr2>",
+		      cmd_bridging_table_add, 6, 0),
+	SHELL_CMD_ARG(table-remove, NULL, "<NetIdx1> <NetIdx2> <Addr1> <Addr2>",
+		      cmd_bridging_table_remove, 5, 0),
+	SHELL_CMD_ARG(subnets-get, NULL, "<Filter> <NetIdx> <StartIdx>", cmd_bridged_subnets_get,
+		      4, 0),
+	SHELL_CMD_ARG(table-get, NULL, "<NetIdx1> <NetIdx2> <StartIdx>", cmd_bridging_table_get,
+		      4, 0),
+	SHELL_SUBCMD_SET_END);
+
+SHELL_SUBCMD_ADD((mesh, models), brg_cfg, &brg_cfg_cmds, "Bridge Configuration Cli commands",
```

Comment:
I would prefer to have it as easy as possible to use this. Typing underscores require an extra step.

```suggestion
SHELL_SUBCMD_ADD((mesh, models), brg-cfg, &brg_cfg_cmds, "Bridge Configuration Cli commands",
```

Reply:
Hmm both sol_pdu_rpl and od_priv_proxy uses underscores, so in this case we should change those as well. Though with tab-completion you won't need to deal with the underscore so I don't really think this is an issue.

Reply:
but sar and rpr use dashes :)

Reply:
Huh ... I guess we need to see how can we improve the patterns. I will submit upstream issue to change this. Let's see.